src/zui/ZUICreatePerson/PersonalInfoForm.tsx

```diff
@@ -0,0 +1,247 @@
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { ExpandMore } from '@mui/icons-material';
+import utc from 'dayjs/plugin/utc';
+import {
+  Box,
+  Button,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+} from '@mui/material';
+import dayjs, { Dayjs } from 'dayjs';
+import { FC, useEffect, useRef, useState } from 'react';
+
+import { checkInvalidFields } from '.';
+import formatUrl from 'utils/formatUrl';
+import globalMessageIds from 'core/i18n/globalMessageIds';
+import InfoInputForm from './InfoInputForm';
+import { makeNaiveDateString } from 'utils/dateUtils';
+import messageIds from 'zui/l10n/messageIds';
+import { TagManagerSection } from 'features/tags/components/TagManager';
+import useCustomFields from 'features/profile/hooks/useCustomFields';
+import { useNumericRouteParams } from 'core/hooks';
+import useTags from 'features/tags/hooks/useTags';
+import { Msg, useMessages } from 'core/i18n';
+import { ZetkinCreatePerson, ZetkinTag } from 'utils/types/zetkin';
+
+dayjs.extend(utc);
+
+type ShowAllTriggeredType = 'enter' | 'mouse' | 'none';
```

Comment:
I think this type would be better as `keyboard` | `mouse` | `null`.

Instead of working just with `enter`, it should also detect other "confirm" keys, e.g. space.

Instead of having a string value (`'none'`)  for the falsy option, use `null` which is already falsy.